### ISSUE TITLE
IOS/Android - Workspace- Google search doesn't work in Information, Company, Personal and  Additional information pages

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#8951]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch the app
2. Log in with expensifail account
3. Go to Setting - Preferences
4. Enable Secure Staging server
5. Go to any Workspace
6. Tab connect Bank account
7. Tab Connect online with Plaid
8. Select Wells Fargo bank account
9. Tap user name: user_good
10. Tap password: pass_good
11. Tap Continue
12. Check the search address on Company, Personal and  Additional information pages

## Expected Result:
Google search works in Information, Company, Personal and  Additional information pages

## Actual Result:
Google search doesn't work in Information, Company, Personal and  Additional information pages

## Workaround:
Uknown

## Platform:

Where is this issue occurring?

- iOS
- Android

**Version Number:** 1.1.57.12

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** 

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation\

https://user-images.githubusercontent.com/93399543/167945161-d96c689f-17b2-4081-bdd7-67dce30e4065.mp4

https://user-images.githubusercontent.com/93399543/167945164-6ded2128-34ff-429d-a543-e58372ba63a8.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kaushiktd** [comment]
Is this issue still available to work on?

**Comment by MariaHCD** [comment]
Can't reproduce the issue due to: https://github.com/Expensify/App/pull/8966#issuecomment-1126263948

Will look into this once that PR is merged

**Comment by MariaHCD** [comment]
Able to reproduce the issue. The search seems to work fine on web.

**Comment by MitchExpensify** [comment]
[Upwork Job ](https://www.upwork.com/jobs/~01e6d247545e4ef309)

**Comment by Luke9389** [comment]
@kbecciv, how do you do step 4 (enable Secure Staging server)?

**Comment by MariaHCD** [comment]
@Luke9389 I believe step 4 refers to navigating to Settings > Preferences and enabling `Use Secure Staging Server`:

![Screen Shot 2022-05-20 at 10 38 08 AM](https://user-images.githubusercontent.com/12268372/169468060-c169a575-f7a4-48f7-b4f2-c9529a3de958.png)

**Comment by allroundexperts** [comment]
Looks like we are using a wrong API key. Did you follow the instructions (here)[https://github.com/FaridSafi/react-native-google-places-autocomplete]?

```
Step 2.

Get your [Google Places API keys](https://developers.google.com/maps/documentation/places/web-service/get-api-key/) and enable "Google Places API Web Service" (NOT Android or iOS) in the console. Billing must be enabled on the account.

Step 3.

Enable "Google Maps Geocoding API" if you want to use GoogleReverseGeocoding for Current Location
```

Screenshot of the error attached.
<img width="388" alt="Screenshot 2022-05-22 at 5 58 07 PM" src="https://user-images.githubusercontent.com/30054992/169696308-5782c0d7-5bde-41bc-8967-a0082bfad6d4.png">

@parasharrajat @MitchExpensify @Luke9389

**Comment by parasharrajat** [comment]
Yeah, this is known to us. It seems like a duplicate issue cc: @Julesssss.

**Comment by allroundexperts** [comment]
Can we put this on hold or something so that people don't spend time on it?

**Comment by parasharrajat** [comment]
It was not supposed to be exported. We will take this down on Monday. There is nothing much an external contributor can do here. cc: @MitchExpensify

**Comment by Julesssss** [comment]
Closing as a [dupe](https://github.com/Expensify/App/issues/8797#issuecomment-1111022077). @MitchExpensify please close out the UpWork job, thanks!

**Comment by MitchExpensify** [comment]
Got it, Upwork job closed @Julesssss

**Comment by Julesssss** [comment]
Thank you kindly
